### DISCUSSION TITLE
It seems that there are no files with a .log extension in the Karavan-app web development???????????

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1257] [Q&A]
### Ask a question

I am not able to see the files related to the logs in the karavan-app . 
<img width="527" alt="image" src="https://github.com/apache/camel-karavan/assets/14121796/9496a067-0ad8-41e0-afc9-15a8229dcf56">

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by kondetiramanakumar** [comment]
@davsclaus Required your inputs for this????

**Comment by davsclaus** [comment]
this is not a karavan end user question - you can build karavan project yourself and configure logging in karavan-app as you like.
https://github.com/apache/camel-karavan/blob/main/karavan-app/src/main/resources/application.properties
